src/app/globals.scss

```diff
@@ -10,11 +10,10 @@ body {
   box-sizing: border-box;
 }
 
-// light mode
-:root[data-theme='light'] {
+:root {
```

Comment:
any reason why the light mode is removed?

Reply:
I assumed light mode would be the default